src/inner-slider.js

```diff
@@ -97,6 +97,11 @@ export class InnerSlider extends React.Component {
         slide.onblur = this.props.pauseOnFocus ? this.onSlideBlur : null;
       }
     );
+
+    if (this.props.touchMove) {
+      this.list.addEventListener("touchmove", this.swipeMove);
```

Comment:
did you remove { passive: false } because it's the default? I'd probably keep it and add a comment to explain that we need passive set to false to allow preventDefault and link to https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener for more details.

Reply:
I felt that it was unnecessary to leave it in, as false is the default. I can add it back in to guard against a future change to the default (although I'd guess it'd be very unlikely to happen)

Reply:
Added it back in and added note